### PR TITLE
[WIP] Proposed Definition: Boogaloo

### PR DESCRIPTION
**Pull request by kathryngraysonnanz**
New PR for Boogaloo, including the Hawaiian shirt / igloo content additions from @ericwbailey (thank you!!!)

### CODE REVIEW

File: 11ty/definitions/boogaloo.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Its usage is derived from memes about the film _Breakin’ 2: Electric Boogaloo_, riffing on the title – notably, as “Civil War 2: Electric Boogaloo”. Appending “Electric Boogaloo” to other concepts as a joke was born in and largely popularized by early internet message boards such as Something Awful and 4chan.
+
+Some Alt-Right members self-identify as "Boogaloo Boys" (or "Bois"), or speak of "preparing / showing up for the Boogaloo". Many dress in Hawaiian shirts and/or military wear to distinguish themselves at protests and gatherings. One effect of wearing the Hawaiian shirt is to co-opt the public’s existing notions about Hawaiian culture with peaceful, impartial observation, and not radical militarism.
+
+## Issues
+
+Similarly to the [OK Hand symbol](/definitions/ok-hand.md), the word itself does not have racist origins (and in fact, is a product of the Latin music community) and neither does the original meme (a joke about movie sequels). However, language evolves, and "Boogaloo" has become a term used quite seriously by Alt-Right and White Supremacist communities.
```
Comment (tatianamac) [inline 11ty/definitions/boogaloo.md]:
```suggestion
Similarly to the [OK Hand symbol](/definitions/ok-hand), the word itself does not have racist origins (and in fact, is a product of the Latinx music community) and neither does the original meme (a joke about movie sequels). However, language evolves, and "Boogaloo" has become a term used quite seriously by Alt-Right and White Supremacist communities.
```

File: 11ty/definitions/boogaloo.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Related symbols
+
+Both the Hawaiian shirt and Igloo snow huts are associated with Indigenous cultures. Before being co-opted by the Boogaloo movement, both symbols were co-opted by American popular culture.
+
+Hawaiian shirts were popularized by the Tiki culture movement, created after American military returned from being stationed in the South Pacific during World War II. Due to a lack of including and elevating Polynesian people in the movement, it is considered to be both cultural appropriation and colonial nostalgia.
+
+## Impact
+
+Although this was previously a non-violent / non-hateful meme, continuing to use it supports the plausible deniability of those who do use it with serious intent – and may put you in danger of being mistaken for a "Boogaloo Boy".
```
Comment (tatianamac) [inline 11ty/definitions/boogaloo.md]: I think the impact here is a bit solely focused on perpetrator and not the harmed.

One aspect of hate symbols being dogwhistled is that those who are most impacted/vulnerable will not know what your intent is and therefore will feel the fear regardless.

**Review by tatianamac — APPROVED** [review]

**Comment by tatianamac** [comment]
@kathryngraysonnanz Thanks for your patience with this! I added a few comments. 🤗

**Comment by kathryngraysonnanz** [comment]
@tatianamac – thank you for the feedback! I incorporated your suggested changes and reworked the Impact section.

**Comment by tatianamac** [comment]
Thank you, @kathryngraysonnanz ! Based on our other convos, I'm going to guess that you'd prefer to be left out of the attribution? (I wanted to make sure to ask either way) Also @ericwbailey, what about you? Would you like to be attributed for your contribution from the Self-Defined account? Thanks to you both!

**Comment by ericwbailey** [comment]
I view this is making some slight edits and additions to @kathryngraysonnanz's existing work. If they prefer not to be included in the attribution, I'm happy to honor that and be left off as well.

Looking forward to submitting my own terms in the future, however!

**Comment by kathryngraysonnanz** [comment]
@tatianamac – thanks for checking! Yeah, considering the subject nature of this particular term, I'd prefer not to be twitter tagged in association with it. Maybe that's over-cautiousness, but I'd rather err on the safer side.

**Pull request merged**